### ISSUE TITLE
Kitsu Not coming online

### ISSUE THREAD

**Issue by krishme** [cgwire/zou#226]
Kitsu goes down after a system reboot and does not come online. I have started zou, zou-events, redis, nignx, postgresql. I am using Ubuntu 18.04 LTS
I get an error message as Kitsu encountered an error while reaching its data API.

Kindly help

**Comment by frankrousseau** [comment]
Hello,

Can you provide us with the result of `curl http://localhost:5000/status` and with the content of `/var/log/zou/gunicorn_error.log`?

**Comment by krishme** [comment]
Hi Frank
Thanks for your reply
http://localhost:5000/status
name | "Zou"
version | "0.11.23"
database-up | true
key-value-store-up | true
event-stream-up | true
gunicorn_error.log is located in /opt/zou/logs/ as per the installation which reads as attached screenshot
![Screenshot from 2020-02-20 10-44-46](https://user-images.githubusercontent.com/59594156/74903105-0a560780-53ce-11ea-8d6f-38653eac473e.png)

**Comment by frankrousseau** [comment]
Is your Nginx server up?

**Comment by krishme** [comment]
Sorry for the delay in reply, I was travelling. 
yes nginx server is up and running, also i checked zou and zou-events both are running (active)

**Comment by frankrousseau** [comment]
Can you send the exact error message you have?

**Comment by krishme** [comment]
I am not getting any error messages when I check gunicorn_error.log(if any other error log file, please let me know, I'll check it) . I had to reinstall cuz we had a management presentation, but again it went down, and its in the same state, unable to fetch the url in the browser. all the services are up.

**Comment by frankrousseau** [comment]
Yes but can you give me the browser error? It looks like it's an error in your proxy.

We can provide cloud trial instance too.

**Comment by ddesmond** [comment]
Did you check your firewall setup? Worth investigating

**Comment by krishme** [comment]
I checked the ufw status - disabled (inactive)

**Comment by krishme** [comment]
Browser - Firefox getting error as below
Firefox can’t establish a connection to the server at localhost.

    The site could be temporarily unavailable or too busy. Try again in a few moments.
    If you are unable to load any pages, check your computer’s network connection.
    If your computer or network is protected by a firewall or proxy, make sure that Firefox is permitted to access the Web. (access to the web is enabled)

**Comment by krishme** [comment]
@frankrousseau 
If our testing succeeds we are planning to do in-house hosting with some customization from CG-Wire
